src/tasks/tests/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks.Tests.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
+    <Nullable>enable</Nullable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildFrameworkVersion)" />
+    <PackageReference Include="Moq" Version="$(MoqVersion)" />
+  </ItemGroup>
```

Comment:
Because of [this](https://github.com/danmoseley/runtime/blob/85078215aa7039da526c993cc8d4f5deaae6cbe9/src/tasks/tasks.proj#L3), this project gets always built, even in an official build which doesn't run tests. We want to minimize the external dependencies that we restore and use during an official build.

I think tasks.proj should be changed so that task tests are only built when requested.

Reply:
What's the right way to do that?

Reply:
You would need to define a new subset in Subsets.props: tasks.tests. Then if that subset is passed in you can pass an additional property to tasks.proj: https://github.com/dotnet/runtime/blob/56ee7c5950b55529c2b5618d8f9b796d1b425813/Build.proj#L15

i.e.
```xml
<Properties Condition="...">IncludeTaskTests=true</Properties>
```

Then in tasks.proj you would change this line: https://github.com/dotnet/runtime/blob/56ee7c5950b55529c2b5618d8f9b796d1b425813/src/tasks/tasks.proj#L3

to something like
```
<ProjectReference Include="$(MSBuildThisFileDirectory)**\*.csproj" />
<ProjectReference Remove="$(MSBuildThisFileDirectory)tests\**\*.csproj" Condition="'$(IncludeTaskTests)' != 'true'" />
```